hibernate-reactive-core/src/main/java/org/hibernate/reactive/persister/collection/impl/ReactiveCollectionPersister.java

```diff
@@ -0,0 +1,65 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive.persister.collection.impl;
+
+import java.io.Serializable;
+import java.util.concurrent.CompletionStage;
+
+import org.hibernate.collection.spi.PersistentCollection;
+import org.hibernate.engine.spi.SharedSessionContractImplementor;
+
+/**
+ * Similar to {@link org.hibernate.persister.collection.AbstractCollectionPersister} in ORM
+ * <p>
+ *     The reactive classes already extend something so we cannot have an abstract class like in ORM.
+ * </p>
+ */
+public interface ReactiveCollectionPersister {
+
+	/**
+	 * Reactive version of {@link org.hibernate.persister.collection.AbstractCollectionPersister}.recreate()
```

Comment:
Is this a common way to link to the javadoc?
I usually do:
```
{@link org.hibernate.persister.collection.AbstractCollectionPersister#recreate}
```
Same for the rest of the javadoc.

Reply:
We could also remove the parameters from the javadoc since they don't have any helpful comment (I don't think they need one).

Reply:
This also applies to the javadoc of the other methods.

Reply:
The best I can tell is:   ```{@link org.hibernate.persister.collection.AbstractCollectionPersister#insertRows()}```

thx

Reply:
As long as it is clickable

Reply:
I can't find a way to create a clickable link to a method.

Best I could do is: ```   * Reactive version of {@link org.hibernate.persister.collection.AbstractCollectionPersister} recreate() method```

Reply:
links to methods need to list the parameter types.